### ISSUE TITLE
Autoload plugin manager

### ISSUE THREAD

**Issue by Mte90** [codicocodes/dotfyle#99]
So In my dotfiles I use vimplug but is autoloaded https://dotfyle.com/Mte90/dotfiles-config-nvim so I don't commit it but I call vimplug on https://github.com/mte90/dotfiles/blob/master/.config/nvim/lua/plugins.lua with the command.
Maybe a way could be scan for that command in the various files?

**Comment by codicocodes** [comment]
Yes currently we are only matching Packer and Lazy, but I definitely intend to add support for vimplug and others as well.
